cmd/kubebuilder/initproject/init.go

```diff
@@ -43,13 +45,21 @@ var domain string
 var copyright string
 var bazel bool
 var controllerOnly bool
+var newVersion bool
 
 func AddInit(cmd *cobra.Command) {
 	cmd.AddCommand(repoCmd)
 	repoCmd.Flags().StringVar(&domain, "domain", "", "domain for the API groups")
 	repoCmd.Flags().StringVar(&copyright, "copyright", filepath.Join("hack", "boilerplate.go.txt"), "Location of copyright boilerplate file.")
 	repoCmd.Flags().BoolVar(&bazel, "bazel", false, "if true, setup Bazel workspace artifacts")
 	repoCmd.Flags().BoolVar(&controllerOnly, "controller-only", false, "if true, setup controller only")
+	repoCmd.Flags().BoolVar(&newVersion, "newkb", false, "if true, init project with kubebuilder 1.0")
```

Comment:
Maybe use `kubebuilder-version` with string type.
Valid value is either `v0` or `v1`.

Reply:
I like @mengqiy suggestion. However it is *more* about the project version instead of kubebuilder version. kubebuilder is going to support both the projects (v0 and v1). So we can name the flag `project-version` of type string with values `v1` or `v0`. `v0` being default.

Reply:
done